lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -152,7 +153,10 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 .TOO_MANY_RETRY_ATTEMPTS);
             }
 
-            ThirdPartyAPIService thirdPartyAPIService = selectThirdPartyAPIService();
+            ThirdPartyAPIService thirdPartyAPIService =
+                    selectThirdPartyAPIService(
+                            configurationService.getUseLegacy() != true,
```

Comment:
Can we change this param so its actually dvaDirectEnabled rather than useLegacy

Reply:
Corrected.